ArraySimulation/MPPT/GlobalMPPTAlgo/VoltageSweep.py

```diff
@@ -0,0 +1,120 @@
+"""
+VoltageSweep.py
+
+Author: Afnan Mir, Array Lead (2021).
+Contact: afnanmir@utexas.edu
+Created: 02/06/2021
+Last Modified: 02/06/2021
+
+Description: The Voltage Sweep class is a concrete class that provides the basis for how a global 
+MPPT algorithm will perform a voltage sweep to find all of its local maxima. It enables us to find
+lower and upper bounds of the global maximum
+
```

Comment:
I suggest having a brief description of the voltage sweep procedure. See MPPTAlgorithm concrete classes for reference.